doc/contributing/coding-conventions.chapter.md

```diff
@@ -670,3 +670,18 @@ stdenv.mkDerivation {
   ...
 }
 ```
+
+### Import From Derivation {#ssec-import-from-derivation}
+
+Import From Derivation (IFD) is disallowed in Nixpkgs for performance reasons:
+[Hydra] evaluates the entire package set, and sequential builds during evalation would increase evaluation times to become impractical.
+
+[Hydra]: https://github.com/NixOS/hydra
+
+Import From Derivation can be worked around in some cases by checking light-weight intermediate build results, such as files containing dependency information, into version control and using them like regular sources.
```

Comment:
I think this obscures more than the original wording.

How about that:

```suggestion
Import From Derivation can be worked around in some cases by checking generated intermediate files into version control and reading those like regular sources instead.
```

Reply:
> by checking

This reads like _checking_ instead of _checking into_.

```suggestion
Import From Derivation can be worked around in some cases by committing generated intermediate files and reading those like regular sources instead.
```

Reply:
```suggestion
Import From Derivation can be worked around in some cases by committing generated intermediate files and reading those instead.
```

To what degree are they like regular sources? "[committed] generated intermediate files" is descriptive enough, without provoking unproductive thoughts about the comparison.

Reply:
Right. I was just stumbling over just saying "commit", because that's jargon and [appears to be non-grammatical](https://dictionary.cambridge.org/dictionary/english/commit) without a target.

```suggestion
Import From Derivation can be worked around in some cases by committing generated intermediate files to version control and reading those instead.
```

Reply:
LGTM!

Explaining commit as jargon shows your commitment to accessibility. :heart: